vxquery-xtest/src/test/resources/ExpectedTestResults/Simple/test.xml

```diff
@@ -0,0 +1 @@
+<stationCollection pageSize="100"pageCount="1"totalCount="1"><station><id>GHCND:US000000001</id><displayName>Station 1</displayName><latitude>10.000</latitude><longitude>-10.000</longitude><elevation>1000.0</elevation><locationLabels><type>ST</type><id>FIPS:1</id><displayName>State 1</displayName></locationLabels><locationLabels><type>CNTY</type><id>FIPS:-9999</id><displayName>County 1</displayName></locationLabels><locationLabels><type>CNTRY</type><id>FIPS:US</id><displayName>UNITED STATES</displayName></locationLabels></station></stationCollection>
```

Comment:
All these test changes for Simple seem to be out of place. Did you add a test to Simple or to FunctionsAndOperatorsOnNumericQueries? Please make sure all the tests in their correct location and you do not have any extra additions in the git change request.